CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+# OpenKit Java Changelog
+
+## [Unreleased](https://github.com/Dynatrace/openkit-java/compare/v1.0.1...HEAD)
+
+### Added
+- Extend OpenKit objects from java.io.Closeable to add try-with-resources statement compatibility  
```

Comment:
Implement java.io.Closeable interface in OpenKit objects to add try-with-resources statement compatibility